app/screens/claim/DateClaim.tsx

```diff
@@ -27,7 +27,7 @@ const DateClaim = ({
         }}
       >
         <Text style={{ color: item.value ? undefined : colors.gray }}>{`${
-          item.value || "Please enter the specific date..."
+          moment(item.value).format('DD/MM/YY') || "Please enter the specific date..."
```

Comment:
Can you double check this?  Need `YYYY`  
![image](https://user-images.githubusercontent.com/8411406/119455581-14a74400-bd7d-11eb-9145-077bbbfa66b2.png)

Reply:
Done